modernomad/core/tasks.py

```diff
@@ -34,23 +34,37 @@ def send_guest_welcome():
     logger.info("Running task: send_guest_welcome")
     # get all bookings WELCOME_EMAIL_DAYS_AHEAD from now.
     locations = Location.objects.all()
+
+    # to ensure tests actually do something
+    did_send_email = False
+
     for location in locations:
         soon = datetime.datetime.today() + datetime.timedelta(days=location.welcome_email_days_ahead)
         upcoming = Use.objects.filter(location=location).filter(arrive=soon).filter(status='confirmed')
         for booking in upcoming:
             guest_welcome(booking)
+            did_send_email = True
```

Comment:
This will get overwritten each time through... so will only give information about the last location in the list. not the end of the world but we might want to build up a list of the success/failure reporting. Happy for it to be a #fixme.

Reply:
The intention is just to report back to the test that an email was sent -- it doesn't need to collect any information for this purpose. The idea is if this task is a no-op for whatever reason (e.g. the date calculation is wonky), the test should fail.